packages/kit/test/apps/basics/src/routes/errors/error-html/make-root-fail/+server.js

```diff
@@ -1,9 +1,7 @@
-import { set_should_fail } from '../../../+layout.server';
-
 /**
  * @type {import('./$types').RequestHandler} param0
  */
-export function GET({ url }) {
-	set_should_fail(url.searchParams.get('type'));
+export function GET({ cookies, url }) {
+	cookies.set('fail-type', url.searchParams.get('type'));
```

Comment:
Haven't tested, but we probably need to set the path attribute here
```suggestion
	cookies.set('fail-type', url.searchParams.get('type'), { path: '/' });
```

Reply:
I suspect it will work because implicitly, the path is `/errors/error-html` which is the path we visit when the cookie gets read, but there's no harm in making it apply globally